Firmware/Marlin_main.cpp

```diff
@@ -8417,6 +8417,18 @@ SERIAL_PROTOCOLPGM("\n\n");
 
 #endif //TMC2130_SERVICE_CODES_M910_M918
 
+    /*!
+	### M919 - Get silent mode status <a href="https://reprap.org/wiki/G-code#M919:_Report_silent_mode_status">M919: Report silent mode status</a>
+	
+    */
+	case 919:
+    {
+#ifdef TMC2130
```

Comment:
It would be better if you moved the `#ifdef TMC2130` to encompass the entire case, so that on a printer without TMC2130 you will get "unknown gcode" or something like that automatically. There's no reason to handle something like this with the MK2.5/S since there the Mode doesn't change the max speed/acceleration